### PR TITLE
Add LFS support

### PR DESCRIPTION
**Pull request by ericgaspar**
#47

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -21,6 +21,7 @@ path_url=$(ynh_normalize_url_path --path_url $(ynh_app_setting_get --app $app --
 db_password=$(ynh_app_setting_get --app $app --key mysqlpwd)
 admin=$(ynh_app_setting_get --app $app --key adminusername)
 key=$(ynh_app_setting_get --app $app --key secret_key)
+lfs_key=$(ynh_app_setting_get --app $app --key lfs_key)
 port=$(ynh_app_setting_get --app $app --key web_port)
```
Comment (Josue-T) [inline scripts/upgrade:25]: You might need to manage the migration if we upgrade from an old version without LFS configured

**Review by Josue-T — CHANGES_REQUESTED** [review]
I'll look like of for except the upgrade from old version. 👍

File: scripts/upgrade [hunk 2]
```diff
@@ -43,6 +40,28 @@ then
...
+ynh_abort_if_errors
+
+#=================================================
+# STOP SYSTEMD SERVICE
+#=================================================
+ynh_script_progression --message="Stopping a systemd service..." --weight=1
+
+# We stop the service before to set ynh_clean_setup
+ynh_systemd_action --service_name=$app --action="stop"
+
```
Comment (Josue-T) [inline scripts/upgrade:53]: Well why do you stop the service after the backup. When you do this you won't be sure that you keep the data integrity when you run the back at the same time that the service is running.

**Review by Josue-T — CHANGES_REQUESTED** [review]

File: scripts/upgrade (hunk 2 above)
Comment (ericgaspar) [reply scripts/upgrade]: You should also express this concerns in the [example_yn](https://github.com/YunoHost/example_ynh/blob/master/scripts/upgrade#L38-L60) repo as it is the recommended order (first backup then stop service)

**Comment by ericgaspar** [comment]
!testme

**Comment by Josue-T** [comment]
Let's merge as the CI issue is not linked to this PR.

**Pull request merged**